db/seeds.rb

```diff
@@ -82,4 +91,9 @@
   names[key][0].save!
 end
 
+# Create news section & home_node
+unless Section.find_by(name: 'news').present?
```

Comment:
Or you can just `Section.find_or_create_by name: 'news'` and skip the conditional...